src/Error/Debugger.php

```diff
@@ -914,6 +914,7 @@ public static function printVar($var, array $location = [], ?bool $showHtml = nu
                 $lineInfo = sprintf('%s (line %s)', $file, $line);
             }
         }
+        /** @psalm-suppress RedundantCondition */
```

Comment:
Sorry about this - it'll be fixed in the next release

Reply:
No problem. BTW can psalm report unneeded ignores?

Reply:
Not at the moment - it's ticketed here: https://github.com/vimeo/psalm/issues/1444

Next release should happen in a few hours

Reply:
Thanks, I'll keep an eye on that issue.

Reply:
Next release (3.2.2) is out. You'll find one annoyance - `src/Core/ObjectRegistry.php:122:9 - Class or interface Cake\Core\InstanceConfigTrait does not exist`

Don't know how you want to handle that - you might suppress it for the moment. I'm interested [in a longer term solution, too](https://github.com/vimeo/psalm/issues/1478).

Reply:
Yup I ignored them for now.